practicals/GNN_for_recommendations.ipynb

```diff
@@ -0,0 +1,1284 @@
+{
```

Comment:
<p>"Since we already have a test set edges via test_mask. We will use this to demonstrate the inference." -&gt; "Since we already have a test set of edges via <code>test_mask</code>, we will use this to demonstrate the inference."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2024/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>

Reply:
<p>fixed :) </p>